src/pseudopeople/configuration/generator.py

```diff
@@ -147,20 +150,16 @@ def _generate_configuration(config_type: str) -> ConfigTree:
     noising_configuration.update(baseline_dict, layer="baseline")
 
     # Update configuration with non-baseline default values
-    if config_type == Keys.DEFAULT:
+    if config_type == DEFAULT:
```

Comment:
Can't we just delete this `DEFAULT` concept? It seems like the only time you wouldn't want to update to the defaults is if you have a `NO_NOISE` config in which case you can just use `if config_type != NO_NOISE` logic. This would then get rid of the confusion above.